### DISCUSSION TITLE
Please Help me to get a github student package

### DISCUSSION THREAD

**Discussion by kemooamego** [community/community#40915] [GitHub Education]
### Select Topic Area

Product Feedback

### Body

I am a student at Delta Higher Institute of Computing in Egypt, and I want to get a GitHub fund. Every time I apply, I get a rejection, and I need this pakage. It says that the problem is that I did not complete all the data related to the profile of the account, and I am sure that I completed all the required data.

**Comment by ossanna16** [comment]
Hi there @kemooamego and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
